### PR TITLE
API for network pool

### PR DESCRIPTION
**Pull request by tshihad**

### CODE REVIEW

File: pkg/common/constants.go [hunk 1]
```diff
@@ -15,6 +15,7 @@ const (
 	NetworksPath = "networks"
 	// NetworkTypePath
 	NetworkTypePath = "network-types"
+	NetworkPoolPath = "pools"
```
Comment (gandharvas) [inline pkg/common/constants.go:18]: Should the CI fail here? or have we removed the CI validation for Commment on exported names?

Comment (tshihad) [reply pkg/common/constants.go:18]: I will go with the second approch. It will be a huge task to comment all exported fields in sdk

Comment (gandharvas) [reply pkg/common/constants.go:18]: yeah, sure. But my concern is did the CI run fine?

**Review by gandharvas — APPROVED** [review]
CI ran fine.

File: pkg/common/constants.go (hunk 1 above)
Comment (tshihad) [reply pkg/common/constants.go:18]: Yeah, we can explicitily specify the CI to skip the checks for exported names

**Review by arunkumarviswanathan — APPROVED** [review]
LGTM

**Pull request merged**